### ISSUE TITLE
Add ESP32 SVD files

### ISSUE THREAD

**Issue by maxgerhardt** [platformio/platform-espressif32#1196]
Espressif has released them at https://github.com/espressif/svd/releases.

**Comment by pwm802** [comment]
Has there been any activity on this?

**Comment by TheHexaCube** [comment]
I'm having the same issue - included the .svd file for an ESP32-C3, yet no data is shown under the peripherals tab. 
Instead I get the error message "Cannot read properties of null (reading 'recordType')"

**Comment by inF1704** [comment]
Same issue here. I tried to include the svd file but without success

**Comment by maxgerhardt** [comment]
The SVD parser of the PlatformIO extension is known to be borked with certain SVDs. Probably other extensions like https://marketplace.visualstudio.com/items?itemName=mcu-debug.peripheral-viewer can handle it though.

https://github.com/platformio/platformio-vscode-ide/issues/1675

**Comment by inF1704** [comment]
Any chance it gets fixed?

**Comment by maxgerhardt** [comment]
Ask the devs, not me- Push the linked issue.

**Comment by Jason2866** [comment]
Looks like Platformio stopped every development work regarding espressif stuff

**Comment by pmendiuk** [comment]
> Looks like Platformio stopped every development work regarding espressif stuff

They stopped working on Espressif in general or just the SVD parser?

**Comment by maxgerhardt** [comment]
Having a fixed SVD parser is also in the interest of other platforms, not just ESP32, so maybe it will get fixed when looking at related SVD parsing bugs.

**Comment by pmendiuk** [comment]
I don't get the impression that register level debugging is a priority, at least for ESP32 users.

**Comment by inF1704** [comment]
I agree with you when using overblown libraries and when the understanding of coding is copy and paste only. But if you need to build a high quality code for performance reqsons, then you build from scratch. For example if you want to interact with an i2C device without using a library you need to debug on register level. Otherwise it is only try and error and because of timings you can't do "debugging" with serial.print

**Comment by pmendiuk** [comment]
@maxgerhardt Do you have any idea if this will ever get fixed/implemented?

**Comment by inF1704** [comment]
Don't know what happened but I managed to establish a debug connection to an esp32s3 with internal JTAG usb debugger and parser running.

![IMG-20241004-WA0017](https://github.com/user-attachments/assets/e6d72977-2db5-45fd-a7c4-d3808e05eb22)
![IMG-20241004-WA0020](https://github.com/user-attachments/assets/e83c9e4f-11af-420c-9056-861ecaf5e576)

**Comment by SherifMEldeeb** [comment]
Is this issue still exist as far as my knowledge it's working now by loading the mentioned svd files to the IDE?!!
